.github/workflows/ci.yaml

```diff
@@ -124,21 +124,17 @@ jobs:
           docker exec -i $CONTAINER_ID ./ci/script.sh
           docker stop $CONTAINER_ID
 
-  ### Step 3: official dockerhub images (bleeding-edge, latest and releases)
-  docker-hub:
+  ### Step 3: official docker image generation
+  pmacct-docker:
     needs: build-and-test
```

Comment:
Do you really want to keep that dependency? I think this stage is something you can do in parallel to testing images and execution, so it can be triggered at the very beginning of this pipeline. But then publish-dockerhub depending on these images generation and the successful tests execution (this build-and-test)

Reply:
I will try to add a commit without this dependency, but I am not sure it will speed up things; you have a number of workers, and these are shared by all "pipelines" (workflows) in github.

Besides, the idea was that - once pmacct has tests - it's not really worth to build some docker containers if tests are not passing, beyond perhaps giving early notice to developers. Thing is, pmacct docker stage is pretty "simple", and it's one of the build combinations already tested in the previous stage, so it's unlikely that it's broken (unless the CI code itself is broken).

I will give it a quick test nevertheless, in case it speeds up things.